basics/data-import/pinot-file-system/import-from-adls-azure.md

```diff
@@ -16,16 +16,14 @@ You can enable the Azure Data Lake Storage using the plugin `pinot-adls`. In the
 By default Pinot loads all the plugins, so you can just drop this plugin there. Also, if you specify `-Dplugins.include`, you need to put all the plugins you want to use, e.g. `pinot-json`, `pinot-avro` , `pinot-kafka-2.0...`
 {% endhint %}
 
-Azure Blob Storage provides the following options -
+Azure Blob Storage provides the following options:
 
 * `accountName` : Name of the azure account under which the storage is created
 * `accessKey` : access key required for the authentication
 * `fileSystemName` - name of the filesystem to use i.e. container name (container name is similar to bucket name in S3)
 * `enableChecksum` - enable MD5 checksum for verification. Default is `false`.
```

Comment:
```suggestion
* `accessKey`: Access key required for the authentication.
* `fileSystemName`: Name of the file system to use, for example, the container name (similar to the bucket name in S3).
* `enableChecksum`: Enable MD5 checksum for verification. Default is `false`.
```

Reply:
This is weird. It's as if half my changes didn't make it into a commit. I need to do some local research to figure out what's going on, because I checked the changes to colons multiple times. Hmm. Again, a second PR will be sent shortly, hopefully this time with all the missing bits.